packages/manifold/contracts/physicalclaim/IPhysicalClaimCore.sol

```diff
@@ -0,0 +1,174 @@
+     *                          what interface
+     * @param amount            (only for ERC1155 tokens) the amount (value) required to burn
+     * @param minTokenId        (only for RANGE validation) the minimum valid token ID
+     * @param maxTokenId        (only for RANGE validation) the maximum valid token ID
+     * @param merkleRoot        (only for MERKLE_TREE validation) the root of the merkle tree of
+     *                          valid token IDs
+     */
+    struct BurnItem {
+        ValidationType validationType;
+        address contractAddress;
+        TokenSpec tokenSpec;
+        BurnSpec burnSpec;
+        uint72 amount;
+        uint256 minTokenId;
+        uint256 maxTokenId;
+        bytes32 merkleRoot;
+    }
+
+    /**
+     * @param id               the ID of the variation
+     * @param maxRedeems       the maximum number of times the variation can be redeemed
+     */
+    struct Variation {
+        uint8 id;
+        uint16 max;
+    }
+
+    /**
+     * @notice a `BurnGroup` is a group of valid `BurnItem`s
+     * @param requiredCount     the number of `BurnItem`s (0 < requiredCount <= items.length) that 
+     *                          need to be included in a burn
+     * @param items             the list of `BurnItem`s
+     */
+    struct BurnGroup {
+        uint256 requiredCount;
+        BurnItem[] items;
+    }
+
+    /**
+     * @notice parameters for burn redeem intialization/updates
+     * @param paymentReceiver   the address to forward proceeds from paid burn redeems
+     * @param totalSupply       the maximum number of redemptions to redeem (0 for unlimited)
+     * @param startDate         the starting time for the burn redeem (0 for immediately)
+     * @param endDate           the end time for the burn redeem (0 for never)
+     * @param burnSet           a list of `BurnGroup`s that must each be satisfied for a burn redeem
+     * @param variations        a list of `Variation`s for the redemptions
+     * @param signer            the address of the signer for the transaction details
+     */
+    struct PhysicalClaimParameters {
+        address payable paymentReceiver;
+        uint32 totalSupply;
+        uint48 startDate;
+        uint48 endDate;
+        BurnGroup[] burnSet;
+        Variation[] variations;
+        address signer;
+    }
+
+    struct PhysicalClaim {
+        address payable paymentReceiver;
+        uint32 redeemedCount;
+        uint32 totalSupply;
+        uint48 startDate;
+        uint48 endDate;
+        BurnGroup[] burnSet;
+        Variation[] variations;
+        address signer;
+    }
+
+    struct Redemption {
+        uint timestamp;
+        uint32 redeemedCount;
+        uint8 variation;
+    }
+
+    struct PhysicalClaimSubmission {
+        uint instanceId;
+        uint32 physicalClaimCount;
+        uint32 currentClaimCount;
+        BurnToken[] burnTokens;
```

Comment:
Move this to end for tight packing.  Reorganize for better packing.

Reply:
Done